soccer/src/soccer/strategy/coach/coach_node.cpp

```diff
@@ -150,48 +150,48 @@ void CoachNode::check_for_play_state_change() {
 }
 
 void CoachNode::assign_positions() {
-    rj_msgs::msg::PositionAssignment positions_message;
-    std::array<uint32_t, kNumShells> positions{};
-    positions[goalie_id_] = Positions::Goalie;
-    if (!possessing_) {
-        // All robots set to defense
-        for (int i = 0; i < kNumShells; i++) {
-            if (i != goalie_id_) {
-                positions[i] = Positions::Defense;
-            }
-        }
-        // Lowest non-goalie robot set to offense
-        if (goalie_id_ == 0) {
-            positions[1] = Positions::Offense;
-        } else {
-            positions[0] = Positions::Offense;
-        }
-    } else {
-        // All robots set to offense
-        for (int i = 0; i < kNumShells; i++) {
-            if (i != goalie_id_) {
-                positions[i] = Positions::Offense;
-            }
-        }
-        // Lowest non-goalie robot set to defense
-        if (goalie_id_ == 0) {
-            positions[1] = Positions::Defense;
-        } else {
-            positions[0] = Positions::Defense;
-        }
-    }
-
-    // Check Overrides
-    if (have_overrides_) {
-        for (int i = 0; i < kNumShells; i++) {
-            if (i != goalie_id_ && current_overrides_[i] == 1 || current_overrides_[i] == 2) {
-                positions[i] = current_overrides_[i];
-            }
-        }
-    }
-
-    positions_message.client_positions = positions;
-    positions_pub_->publish(positions_message);
+    // rj_msgs::msg::PositionAssignment positions_message;
```

Comment:
why is this all commented out?

Reply:
The coach node was overriding the default position assignments so I commented this out for testing.  Now that everything works I'm going to un-comment this, but then the visual test case won't work.  I'll also get rid of offense defaulting to passing when the pr is approved.

Reply:
can you actually check that your code works properly with the new position assignment UI tab? based on the on-field testing today (4/2) there may be some issues, though that also might be due to a variety of other factors (external ref, bad day, etc.)